### PR TITLE
Lab01 uf2 and fix

### PR DESCRIPTION
**Pull request by DanutAldea**
### Pull Request Overview

* Added `uf2` files.
* Added links.
* Fixed code broken import

### Author

Signed-off-by: Aldea George Danut <danutz.aldea23@gmail.com>

### CODE REVIEW

**Review by irina-nita — DISMISSED** [review]

File: docs/lab/01/assets/firmware/pico_h.uf2
Comment (alexandruradovici) [inline docs/lab/01/assets/firmware/pico_h.uf2:1]: this needs to go in `/assets` (in the root folder)

File: docs/lab/01/index.md [hunk 1]
```diff
@@ -496,11 +496,15 @@ For a better understanding, please read [Getting Started in KiCad](https://docs.
 ## Exercises
 
 1. Complete the given exercise sheet.(**3p**)
-2. The microcontroller is already running a predefined program. Assemble a circuit surrounding the microcontroller in order to perform the next tasks:(**3p**)
+<!-- <a target="_blank" href={ require("./assets/firmware/pico_w.uf2").default } download>working</a> -->
+
+1. The microcontroller will be running these precompiled programs for [Pico W](assets/firmware/pico_w.uf2) and [Pico H](assets/firmware/pico_h.uf2). You will need to connect the boards to your PC while holding down the `BOOTSEL` button. This should *"mount"* it as a an external drive. The next step is to copy and paste the `.uf2` files in the `RPI-RP2` drive. Then, assemble a circuit surrounding the microcontroller in order to perform the next tasks:(**3p**)
```
Comment (alexandruradovici) [inline docs/lab/01/index.md:501]:
```suggestion
1. The microcontroller will be running these precompiled programs for [Pico W](/firmware/pico_w.uf2) and [Pico H](/firmware/pico_h.uf2). You will need to connect the boards to your PC while holding down the `BOOTSEL` button. This should *"mount"* it as a an external drive. The next step is to copy and paste the `.uf2` files in the `RPI-RP2` drive. Then, assemble a circuit surrounding the microcontroller in order to perform the next tasks:(**3p**)
```

File: docs/lab/01/index.md [hunk 2]
```diff
@@ -496,11 +496,15 @@ For a better understanding, please read [Getting Started in KiCad](https://docs.
...
 - A status LED is ON as long as the microcontroller is supplied. The pin that determines the status of the LED is GP1.
 - A button is used to toggle the LED integrated on the development board. The pin reading the state of the button is GP2.
 - On the screen is displayed an analog value read by the microcontroller on pin ADC0. Create a circuit in order to display the value $$1.65V$$ on the screen.
 
+If you are curious, you can find the source code of the firmware running [here](https://github.com/UPB-FILS-MA/upb-fils-ma.github.io/tree/main/firmware/lab01).
```
Comment (alexandruradovici) [inline docs/lab/01/index.md:506]:
```suggestion
If you are curious, you can find the source code of the firmware running in the [repository](https://github.com/UPB-FILS-MA/upb-fils-ma.github.io/tree/main/firmware/lab01).
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Comment by alexandruradovici** [comment]
Rebase

File: docs/lab/01/assets/firmware/pico_h.uf2
Comment (DanutAldea) [reply docs/lab/01/assets/firmware/pico_h.uf2:1]: The root folder of the repository?

File: assets/lab01_firmware/pico_h.uf2
Comment (alexandruradovici) [inline assets/lab01_firmware/pico_h.uf2:1]: assets/firmware/lab01/pico_h.uf2

File: docs/lab/01/assets/firmware/pico_h.uf2
Comment (alexandruradovici) [reply docs/lab/01/assets/firmware/pico_h.uf2:1]: yes

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-nita — APPROVED** [review]

**Review by ZamfirAndreiPaul — APPROVED** [review]

**Pull request merged**